modules/assimp/editor_scene_importer_assimp.cpp

```diff
@@ -122,22 +111,23 @@ Node *EditorSceneImporterAssimp::import_scene(const String &p_path, uint32_t p_f
 	//}
 
 	importer.SetPropertyInteger(AI_CONFIG_PP_SBP_REMOVE, aiPrimitiveType_LINE | aiPrimitiveType_POINT);
+
 	//importer.SetPropertyFloat(AI_CONFIG_PP_DB_THRESHOLD, 1.0f);
 	int32_t post_process_Steps = aiProcess_CalcTangentSpace |
+								 aiProcess_GlobalScale | // imports models and listens to their file scale for CM to M conversions
 								 //aiProcess_FlipUVs |
-								 //aiProcess_FlipWindingOrder |
+								 aiProcess_FlipWindingOrder | // very important for culling so that it is done in the correct order.
 								 //aiProcess_DropNormals |
 								 //aiProcess_GenSmoothNormals |
-								 aiProcess_JoinIdenticalVertices |
+								 //aiProcess_JoinIdenticalVertices |
 								 aiProcess_ImproveCacheLocality |
-								 aiProcess_LimitBoneWeights |
 								 //aiProcess_RemoveRedundantMaterials | // Causes a crash
-								 aiProcess_SplitLargeMeshes |
+								 //aiProcess_SplitLargeMeshes |
 								 aiProcess_Triangulate |
 								 aiProcess_GenUVCoords |
 								 //aiProcess_FindDegenerates |
-								 aiProcess_SortByPType |
-								 aiProcess_FindInvalidData |
+								 //aiProcess_SortByPType |
+								 // aiProcess_FindInvalidData |
```

Comment:
This allows invalid files even though they're broken according to Assimp. This may cause previously acceptable files to be rejected and etc.

Reply:
I have enabled aiProcessFindInvalid data but sort by p type is not possible to enable right now as it would need lots of testing.

Reply:
Sounds good. The sort by ptype was to remove vertex colors.